.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @djc @Ralith
+* @djc @Ralith @gretchenfrage
 /quinn-udp @djc @Ralith @mxinden
```

Comment:
Do we need to modify line 2 as well, or is it redundant as-is?

Reply:
I think it might be redundant as-is?

Reply:
GitHub's official example seems to indicate it's redundant as-is. https://github.com/github/docs/blob/main/content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md#example-of-a-codeowners-file

Reply:
Cool, we can fix that in a followup then.